packages/app/src/resource/BotEditor.tsx

```diff
@@ -74,21 +76,21 @@ export function BotEditor(): JSX.Element | null {
       setLoading(true);
       try {
         const code = await getCode();
+        const codeOutput = await getCodeOutput();
         const sourceCode = await medplum.createAttachment(code, 'index.ts', 'text/typescript');
-        const operations: PatchOperation[] = [];
-        if (bot?.sourceCode) {
```

Comment:
Not necessary to check if field is populated and conditionally do a `replace`, since `add` means "add or replace if exists":
https://www.npmjs.com/package/rfc6902#:~:text=add%3A%20Insert%20the%20given%20value%20at%20path.%20Or%20replace%20it%2C%20if%20it%20already%20exists.